config/sidekiq.yml

```diff
@@ -9,6 +9,9 @@
   scheduled_statuses_scheduler:
     every: '5m'
     class: Scheduler::ScheduledStatusesScheduler
+  trending_tags_scheduler:
+    every: '5m'
+    class: Scheduler::TrendingTagsScheduler
```

Comment:
I'm not sure how much expensive the whole thing would be, I'm afraid that running it every 5 minute might be too expensive.

Reply:
It takes milliseconds to run. While the usage buckets are daily, we *do* have an interest in picking up trends as close to real-time as possible, because that's when they're at their most interesting.

Reply:
ok, fine then